scalyr_agent/agent_main.py

```diff
@@ -1949,6 +1984,14 @@ def stop(self):
         default=False,
         help="For status command, prints detailed information about running agent.",
     )
+    parser.add_option(
+        "-H",
+        "--health_check",
+        action="store_true",
+        dest="health_check",
+        default=False,
+        help="For status command, prints health check status. Return code will be 0 for a passing check, and 1 for failing",
```

Comment:
What happens if the health check option is not enabled for the agent process?  We should print a helpful message when they invoke `scalyr-agent-2 status -h`.